pipeline_dp/private_beam.py

```diff
@@ -103,6 +103,35 @@ def expand(self, pcol: pvalue.PCollection) -> pvalue.PCollection:
         return dp_engine.aggregate(pcol, params, data_extractors)
 
 
+class Count(PrivatePTransform):
+    """Transform class for performing DP Count on PrivatePCollection."""
+
+    def __init__(self, count_params: aggregate_params.CountParams, label: Optional[str] = None):
+        super().__init__(return_anonymized=True, label=label)
+        self._count_params = count_params
+
+    def expand(self, pcol: pvalue.PCollection) -> pvalue.PCollection:
+        beam_operations = pipeline_dp.BeamOperations()
```

Comment:
Nit: beam_operations -> ops (yesterday I've renamed it in Sum).

Intelij can do renaming from right click menu on variable name